MetadataExtractor/Formats/Photoshop/Subpath.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Drew Noakes and contributors. All Rights Reserved. Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace MetadataExtractor.Formats.Photoshop
+{
+    /// <summary>
+    /// Represents a subpath created by Photoshop:
+    /// <list type="bullet">
+    /// <item>Closed Bezier knot, linked</item>
+    /// <item>Closed Bezier knot, unlinked</item>
+    /// <item>Open Bezier knot, linked</item>
+    /// <item>Open Bezier knot, unlinked</item>
+    /// </list>
+    /// </summary>
+    /// <author>Payton Garland</author>
+    /// <author>Kevin Mott https://github.com/kwhopper</author>
+    public class Subpath
+    {
+        private List<Knot> p_knots = new List<Knot>();
+
+        public Subpath() : this("")
+        {
+        }
+
+        public Subpath(string type)
```

Comment:
You can merge these two constructors and make the parameter optional, eg:

```suggestion
        public Subpath(string type = "")
```